SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -157,10 +162,17 @@ if [ -n "${CURRENT_ENV_NAME}" ]; then
     exit 1
 fi
 
-# Create environment (acc-py or generic)
+# Create environment (acc-py, conda or generic as default)
 if [ -n "$ACCPY_VERSION" ]; then
-    source $ACCPY_PATH/base/${ACCPY_VERSION}/setup.sh
-    acc-py venv ${ENV_PATH} | tee -a "${LOG_FILE}"
+    source "${ACCPY_PATH}/base/${ACCPY_VERSION}/setup.sh"
+    acc-py venv ${ENV_PATH} | tee -a ${LOG_FILE}
+elif [ -n "$CONDA_ENV" ]; then
+    _log "Creating environment ${ENV_NAME} using Mamba Conda with requirements from (${REQ_PATH})..."
+    mamba env create -f ${REQ_PATH} -n ${ENV_NAME} -y | tee -a ${LOG_FILE}
```

Comment:
Could we defer the installation of the packages to later? Here we just create an empty environment and we add the packages later. This would make it symmetrical with the other builders.